hexrd/ui/image_mode_widget.py

```diff
@@ -408,6 +415,17 @@ def on_eta_max_changed(self, max_value):
                 HexrdConfig().set_polar_res_eta_min(min_value, rerender=False)
         HexrdConfig().polar_res_eta_max = max_value
 
+    def show_overlay_manager(self):
+        if hasattr(self, '_overlay_manager'):
+            self._overlay_manager.ui.reject()
+            del self._overlay_manager
+
+        self._overlay_manager = AzimuthalOverlayManager(self.ui)
+        self._overlay_manager.show()
+
+    def toggle_legend(self, value):
+        HexrdConfig().azimuthal_legend = value
+        HexrdConfig().azimuthal_overlay_modified.emit()
```

Comment:
Can we move the legend and the button to save the plot into the overlay manager? We would need to create a section in that dialog for global options (as opposed to the overlay-specific options displayed at the bottom).